frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -84,14 +84,26 @@ end
 function ReaderCoptListener:updatePageInfoOverride(pageno)
     pageno = pageno or self.ui.view.footer.pageno
 
-    if not (self.document.configurable.status_line == 0 and self.view.view_mode == "page" and self.page_info_override) then
+    local page_info_override = self.page_number == 1 or self.page_count == 1 or self.reading_percent == 1
+        or (self.timer_value == 1 and self.ui.readtimer and self.ui.readtimer:scheduled())
```

Comment:
The idea with this being computed elsewhere was to not have all these checks done on each page turn.
Now it is done even for people who don't enable the top status bar :/

Reply:
Which means 4 conditions checked when none is true :)
At worse, these checks should be after the ones on the next line that check if the top status bar is enabled.